pyccl/halos/profiles.py

```diff
@@ -312,28 +472,33 @@ def _fftlog_wrap(self, cosmo, k, M, a, mass_def,
         plaw_index = self._get_plaw_fourier(cosmo, a)
 
         # Compute Fourier profile through fftlog
-        k_arr, p_fourier_M = _fftlog_transform(r_arr, p_real_M,
-                                               3, 0, plaw_index)
+        k_arr, p_fourier_M = _fftlog_transform(r_arr,
+                                               p_real_M, 3, 0, plaw_index)
         lk_arr = np.log(k_arr)
 
         for im, p_k_arr in enumerate(p_fourier_M):
             # Resample into input k values
-            p_fourier = resample_array(lk_arr, p_k_arr, lk_use,
-                                       self.precision_fftlog['extrapol'],
-                                       self.precision_fftlog['extrapol'],
-                                       0, 0)
+            p_fourier = resample_array(
+                lk_arr,
+                p_k_arr,
+                lk_use,
+                self.precision_fftlog["extrapol"],
+                self.precision_fftlog["extrapol"],
+                0,
+                0,
+            )
             p_k_out[im, :] = p_fourier
         if fourier_out:
-            p_k_out *= (2 * np.pi)**3
+            p_k_out *= (2 * np.pi) ** 3
```

Comment:
I'd remove the space padding of the power operator here, to be consistent with the rest of the code.